spec/controllers/api/v1/conferences_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+describe Api::V1::ConferencesController do
+  let!(:conf_one) { create(:conference, short_title: 'conf_one') }
+  let!(:conf_two) { create(:conference, short_title: 'conf_two') }
+
+  describe 'GET #index' do
+    it 'returns all conferences' do
+
+      get :index, format: :json
```

Comment:
Can you move this and the next line in a before block and then make three test cases for your test?